src/modules/launcher/Wox/ViewModel/ResultsViewModel.cs

```diff
@@ -70,7 +70,7 @@ public ResultViewModel SelectedItem
         }
 
         public Thickness Margin { get; set; }
-        public Visibility Visbility { get; set; } = Visibility.Collapsed;
+        public Visibility Visbility { get; set; } = Visibility.Hidden;
```

Comment:
Was this the reason for `Alt+Space` not working the first time?

Reply:
No. That was happening when we are trying to set focus on textbox. The change here reduces initial loading time for list view.